account_invoice_currency/views/account_invoice.xml

```diff
@@ -36,6 +36,10 @@
         </field>
         <field name="residual_signed" position="attributes">
             <attribute name="sum"/>
+            <attribute name="invisible">True</attribute>
```

Comment:
I'm thinking that we should keep both fields, because both information is interesting.